### PR TITLE
Drag to reorder columns in column editor

### PR DESCRIPTION
**Pull request by MattPutnam**
Columns can now be reordered in the column editor added for enabling/disabling columns.  This uses the same properties as enable/disable to specify which columns can be reordered.

The hamburger icon is a placeholder until someone with better graphix skillz can make us a better one.

### CODE REVIEW

**Comment by eddiebroad** [comment]
I checkout out, compiled, and viewed the code locally and see no reason why it shouldn't be merged.

File: src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs [hunk 1]
```diff
@@ -218,13 +218,31 @@
...
-      (let [style {:padding "4px 8px" :marginRight 5 :borderRadius 5 :cursor "pointer"
+                    :padding "1em" :lineHeight "1.5em" :cursor (when (:drag-active @state) "ns-resize")}
+            :onMouseMove (when (:drag-index @state)
+                           (fn [e]
+                             (let [x (.-clientX e)
+                                   y (.-clientY e)
+                                   dist (utils/distance (:start-x @state) (:start-y @state) x y)
+                                   div-locs (map
+                                              (fn [i] {:index i :y
+                                                       (-> (@refs (str "div" i)) .getDOMNode .getBoundingClientRect .-top)})
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs]: Line lengths across PR.

**Comment by dmohs** [comment]
Works great! Please do fix the line lengths, but otherwise good to merge.

**Comment by MattPutnam** [comment]
I see the one on line 231, but all others are just barely under the limit, unless my settings are wrong.  Which other lines are you seeing?

**Pull request merged**